interested-parties.md

```diff
@@ -35,6 +35,7 @@ Finch | Mark Beacom
 GitHub | Chris Patterson
 GitLab | William Chia
 GitLab | Viktor Nagy
+GitLab | Abuabkar Siddiq Ango | @abuango
```

Comment:
```suggestion
GitLab | Abuabkar Siddiq Ango | abuango
```
@abuango Above suggestion to change from `@abuango` to `abuango` is for row 38.

In addition, in rows 4 and 5, please add additional columns:
```
Organization | Person | GitHub Username
-- | -- | --
```

Reply:
Thanks, @lloydchang. Updated